src/main/scala/higherkindness/rules_scala/workers/zinc/compile/AnalysisStore.scala

```diff
@@ -105,20 +110,122 @@ class AnxAnalyses(format: AnxAnalysisStore.Format) {
   private[this] val reader = new ProtobufReaders(mappers.getReadMapper, schema.Version.V1)
   private[this] val writer = new ProtobufWriters(mappers.getWriteMapper)
 
+  def sortAnnotation(annotation: Annotation): Annotation = {
+    annotation.copy(
+      arguments = annotation.arguments.sortWith(_.hashCode() > _.hashCode())
+    )
+  }
+
+  def sortTypeParameter(typeParameter: TypeParameter): TypeParameter = {
+    typeParameter.copy(
+      annotations = typeParameter.annotations.map(annotation => sortAnnotation(annotation)).sortWith(_.hashCode() > _.hashCode()),
+      typeParameters = typeParameter.typeParameters.map(_typeParameter => sortTypeParameter(_typeParameter)),
+    )
+  }
+
+  def sortStructure(structure: Structure): Structure = {
+    structure.copy(
+      parents = structure.parents.sortWith(_.hashCode() < _.hashCode()),
+      declared = structure.declared.sortWith(_.hashCode() < _.hashCode()),
+      inherited = structure.inherited.sortWith(_.hashCode() < _.hashCode()),
+    )
+  }
+
+  def sortClassLike(classLike: ClassLike): ClassLike = {
```

Comment:
Do we need to sort the repeated `Annotation` here?

Reply:
Yes